src/inc/til/u8u16convert.h

```diff
@@ -145,9 +145,13 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
                             {
                                 // We use our own algorithm because IsDBCSLeadByteEx() supports only a subset of DBCS codepages.
                                 const auto uCh{ gsl::narrow_cast<byte>(ch) };
-                                for (size_t idx{}; gsl::at(cpInfo.LeadByte, idx) != 0; idx += 2) // OK because the LeadByte array is guaranteed to end with two 0 bytes.
+#pragma warning(push)
+#pragma warning(suppress : 26482) // Suppress bounds.2 check for indexing with constant expressions.
+#pragma warning(suppress : 26446) // Suppress bounds.4 check for subscript operator.
+                                for (int idx{}; cpInfo.LeadByte[idx] != 0; idx += 2) // OK because the LeadByte array is guaranteed to end with two 0 bytes.

```

Comment:
FWIW: this is what `til::at` is for :smile: if it can be used here

Reply:
I already tried it and failed to build. But maybe I did something wrong. I was never expecting it was so hard to work with a simple array 😅

Reply:
Haha. I commit one more, go to bed, and see tomorrow what happened...

Reply:
Hm, x86 build this time 🙃 I wait for @miniksa's review before I try to figure out why this fails. Probably I have to do more when I get feedback anyway .